### PR TITLE
New issue templates

### PR DESCRIPTION
**Pull request by ajredniwja**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

1. To make sure we include your contribution in the release notes, please make sure to add description entry for your changes in the "unreleased changes" section of the `CHANGELOG.md` file (at corresponding gem). For the description entry, please make sure it lives in one line and starts with `Feature` or `Issue` in the correct format.

2. For generated code changes, please checkout below instructions first:
  https://github.com/aws/aws-sdk-ruby/blob/master/CONTRIBUTING.md

Thank you for your contribution!

### CODE REVIEW

**Comment by ajredniwja** [comment]
> Could you remove the .idea/ folder files? Adding a .idea to gitignore would also be nice.

Deleted those, will delete ```.github/ISSUE_TEMPLATE.md``` and add .idea to gitignore in a different PR.

**Comment by ajredniwja** [comment]
> Could you remove the .idea/ folder files? Adding a .idea to gitignore would also be nice.

I guess it just picked up my new changes from the repo so there it goes.

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug report
```
Comment (mullermp) [inline .github/ISSUE_TEMPLATE/---bug-report.md]: Is this label correct? I thought it was only called 'bug' .. also I see in one of the templates, this value is in quotes. Does that matter at all?

Comment (ajredniwja) [reply .github/ISSUE_TEMPLATE/---bug-report.md]: I have created one

**Pull request merged**